build-tools/packages/build-infrastructure/src/config.ts

```diff
@@ -13,32 +13,32 @@ import {
 } from "./types.js";
 
 /**
- * The version of the fluidRepo configuration currently used.
+ * The version of the BuildProject configuration currently used.
  */
-export const FLUIDREPO_CONFIG_VERSION = 1;
+export const BUILDPROJECT_CONFIG_VERSION = 1;
 
 /**
- * Top-most configuration for repo layout settings.
+ * Top-most configuration for BuildProject settings.
  */
-export interface IFluidRepoLayout {
+export interface BuildProjectConfig {
```

Comment:
should the interface name be consistent with the other configs and be called `IBuildProjectConfig`?

Reply:
I considered that, but in this case it's not an interface intended to be implemented by anyone outside the package. It's just the "shape" of the config object. So I think the I prefix just gets in the way.